client/index.html

```diff
@@ -214,12 +214,26 @@ <h1 class="title">Settings</h1>
 							<div class="col-sm-12">
 								<h2>Visual Aids</h2>
```

Comment:
Thanks for the precision! I would use the word "Theme" though, or "Interface" maybe, but I prefer "Theme". And we'll include there everything related to design/look options.
What do you think?